src/test/java/com/uber/cadence/activity/ActivityOptionsTest.java

```diff
@@ -150,7 +147,6 @@ public void testBothPresent() throws NoSuchMethodException {
                     .setMaximumInterval(Duration.ofDays(3))
                     .setExpiration(Duration.ofDays(365))
                     .setInitialInterval(Duration.ofMinutes(12))
-                    .setMinimumAttempts(2)
                     .build())
             .build();
     ActivityMethod a =
```

Comment:
I'm not able to leave comments below this line, but should we also get MethodRetry and pass it into merge in the final assertion? i.e.
```
Method method = ActivityOptionsTest.class.getMethod("activityAndRetryOptions");
ActivityMethod a = method.getAnnotation(ActivityMethod.class);
MethodRetry r = method.getAnnotation(MethodRetry.class);
Assert.assertEquals(o, ActivityOptions.merge(a, r, o));
```